rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -112,6 +116,20 @@ bool gui_application::Init()
 		return false;
 	}
 
+#ifdef _WIN32
+	if (m_gui_settings->GetValue(gui::m_attachCommandLine).toBool())
+	{
+		if (AttachConsole(ATTACH_PARENT_PROCESS) || AllocConsole())
+		{
+			[[maybe_unused]] const auto con_out = freopen("CONOUT$", "w", stderr);
+		}
+	}
+	else
+#endif
+	{
+		m_gui_settings->SetValue(gui::m_attachCommandLine, false);
```

Comment:
It doesn't really make sense  to have this as an else path in windows

Reply:
it creates the entry, gui_save struct is not self-registering.